ex3/ex3.c

```diff
@@ -1,15 +1,25 @@
-// Write another program using `fork()`. The child process should print "hello";
-// the parent process should print "goodbye". You should ensure that the child
-// process always prints first.
-
-#include <stdio.h>
-#include <stdlib.h>
-#include <unistd.h>
-#include <sys/wait.h>
-
-int main(void)
-{
-    // Your code here
-
-    return 0;
-}
+// Write another program using `fork()`. The child process should print "hello";
+// the parent process should print "goodbye". You should ensure that the child
+// process always prints first.
+
+// https://repl.it/@codejoncode/Lambda-Excercise-3
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/wait.h>
+
+int main(void)
+{
+    pid_t pid = fork(); 
+
+    if(pid == 0){
+      printf("hello\n");
+    } else{
+      wait(NULL);

```

Comment:
if you want to wait for a specific child, you could also do something like `waitpid(pid, NULL, 0);`